src/ft2font.cpp

```diff
@@ -643,16 +628,14 @@ bool FT2Font::load_char_with_fallback(FT2Font *&ft_object_with_glyph,
                                       bool override = false)
 {
     FT_UInt glyph_index = FT_Get_Char_Index(face, charcode);
-
+    family_names.push_back(face->family_name);
     if (glyph_index || override) {
-        charcode_error = FT_Load_Glyph(face, glyph_index, flags);
-        if (charcode_error) {
+        if ((charcode_error=FT_Load_Glyph(face, glyph_index, flags))) {
```

Comment:
This change seems extraneous; I don't generally like assignments hidden in conditions, especially since we aren't trying to scope the variable to the block.

The spacing around the `=` is missing too.

Reply:
This variable isn't scoped to the block; it's declared as a function parameter passed-by-reference, so the value is actually exported out of the function.